src/main/java/utils/JSActions.java

```diff
@@ -0,0 +1,13 @@
+package utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class JSActions {
```

Comment:
Please create the constructor for this class and store WebDriver as a field here.

Reply:
Created